src/utils/player-error.js

```diff
@@ -1,38 +1,802 @@
 //@flow
-export default class PlayerError {
-  static TYPE: {[name: string]: Object} = {
-    NOT_REGISTERED_PLUGIN: {
-      name: "PluginNotRegisteredException",
-      message: function (name) {
-        return `Cannot load ${name} plugin. Name not found in the registry`;
-      }
-    },
-    NOT_VALID_HANDLER: {
-      name: "PluginHandlerIsNotValidException",
-      message: function () {
-        return "To activate plugin you must provide a class derived from BasePlugin";
+
+/**
+ * @classdesc This is a description of the MyClass class.
+ */
+export default class PlayerError{
```

Comment:
Let's go with the simple Error class name as it encapsulates any kind of error anyway

Reply:
K.